dotcom-rendering/src/components/FrontsAdSlots.tsx

```diff
@@ -67,6 +67,7 @@ export const MerchandisingSlot = ({
 				padSides={false}
 				showTopBorder={false}
 				showSideBorders={false}
+				shouldCenter={false}
```

Comment:
The center styles are now applied to the ad container so they can be overriden for full-width ads